### PR TITLE
Install-ring: Refactor packages installation

### PR DESCRIPTION
**Pull request by JordanP**
Try to use the same code for both Ubuntu and Centos

### CODE REVIEW

**Comment by JordanP** [comment]
Test passed here : https://37.187.159.67:5443/job/ring-install-test/30/

File: jenkins/ring-install.sh [hunk 1]
```diff
@@ -163,15 +164,11 @@ EOF
...
-function _install_node_packages_centos {
-    sudo yum install -y scality-node scality-sagentd scality-nasdk-tools
-}
-
 function _install_base_scality_node_centos {
     _create_credentials_file
     _how_sould_that_be_called
     _install_dependencies_centos
-    _install_node_packages_centos
+    $PKG_INSTALL scality-node scality-sagentd scality-nasdk-tools
```
Comment (cazino) [inline jenkins/ring-install.sh]: I don't think in-lining that is a good idea, it makes the code less readable.

File: jenkins/ring-install.sh [hunk 2]
```diff
@@ -200,17 +197,13 @@ function _configure_sagentd {
...
-}
-
 function _install_base_scality_node_ubuntu {
     # See http://docs.scality.com/display/R43/Setting+Up+Credentials+for+Ring+4.3
     _create_credentials_file
     _how_sould_that_be_called
     _install_dependencies_ubuntu
     _configure_nodes_packages_ubuntu
-    _install_node_packages_ubuntu
+    $PKG_INSTALL -q scality-node scality-sagentd scality-nasdk-tools
```
Comment (cazino) [inline jenkins/ring-install.sh]: Same remark than before on 'in-lining'.

**Comment by cazino** [comment]
LGTM

**Pull request merged**